### ISSUE TITLE
[Profiler][Forms] Exception when two identical forms are rendered on the same page

### ISSUE THREAD

**Issue by vudaltsov** [symfony/symfony#20874]
Symfony 3.2

I have a system of widgets submitted via Ajax. Sometimes I can have the same form rendered twice on the same page by the same embedded controller.

When this happens, the profiler doesn't start correctly, throws a js alert with 404 and "Exception in FormDataCollector.php line 274: Serialization of 'Closure' is not allowed" is shown right at the end of the page.

For right now fixed it with
```php
$this->get('form.factory')->createNamed('form'.random_int(1, 100), ...);
```

**Comment by javiereguiluz** [comment]
@vudaltsov thanks for reporting this issue ... but this is too a duplicated of another reported issue: #20698  This is now fixed in the repository and it'll be available in the upcoming 3.2.1 version. Thanks!

**Comment by vudaltsov** [comment]
I am sorry, next time will try my best not to duplicate things.

12 дек. 2016 г. 3:15 ПП пользователь "Javier Eguiluz" <
notifications@github.com> написал:

Closed #20874 <https://github.com/symfony/symfony/issues/20874>.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/symfony/symfony/issues/20874#event-890270374>, or mute
the thread

**Comment by vudaltsov** [comment]
Thank you very much for help!

12 дек. 2016 г. 3:15 ПП пользователь "Javier Eguiluz" <
notifications@github.com> написал:
